doc/releases/changelog-dev.md

```diff
@@ -303,6 +303,8 @@
 
 <h3>Documentation 📝</h3>
 
+* Part of the documentation is updated to construct `QuantumTape` upon initialization instead of with queuing.
```

Comment:
```suggestion
* The documentation is updated to construct `QuantumTape` upon initialization instead of with queuing.
  [(#4243)](https://github.com/PennyLaneAI/pennylane/pull/4243)
```